### ISSUE TITLE
JAX2TF Keras training limits

### ISSUE THREAD

**Issue by bhack** [google/jax#13329]
### Description

I still see the claim
> We do not yet train the SavedModel, due to b/123499169. 

https://github.com/google/jax/blob/108bc83520a5f107affd6fb9e356f84a55ea93a6/jax/experimental/jax2tf/examples/keras_reuse_main.py#L50-L52

But I don't see anymore the old limits in:
https://github.com/google/jax/blob/main/jax/experimental/jax2tf/examples/saved_model_lib.py

After:
https://github.com/google/jax/commit/b59db5b181c374291f5a72cf9e3ca518fd61b50d

Is this claim still valid can we functionally set `trainable=True`?

### What jax/jaxlib version are you using?

master

**Comment by gnecula** [comment]
Yes, we can set trainable=True now, the custom gradient bug has been fixed.
